mkt/tags/models.py

```diff
@@ -57,3 +56,23 @@ class AddonTag(ModelBase):
 
     class Meta:
         db_table = 'users_tags_addons'
+
+
+def attach_tags(objs, m2m_name=None):
+    """
+    Fetch tags from `objs` in one query and then attach them to a property on
+    each instance. The name of the property will be `m2m_name` + '_list'.
+
+    Assumes every instance in `objs` uses the same model, and needs `m2m_name`
+    to be set to the name of the m2m field between the model and Tag.
+    """
+    if m2m_name is None:
```

Comment:
Setting a default doesn't make something a keyword argument. So I don't see the point in setting it to default `None` just to raise an error if it isn't set.

It isn't a big deal, I just figured if we want to stick to the one positional argument thing a decorator to enforce it is easy enough to write.